users/clients/api/requests.go

```diff
@@ -134,15 +145,13 @@ func (req updateClientOwnerReq) validate() error {
 	return nil
 }
 
-type updateClientCredentialsReq struct {
-	token     string
-	id        string
-	Identity  string `json:"identity,omitempty"`
-	OldSecret string `json:"old_secret,omitempty"`
-	NewSecret string `json:"new_secret,omitempty"`
+type updateClientIdentityReq struct {
+	token    string
```

Comment:
add struct tags

Reply:
`token` is not exported and it is decode from the header not request body